index.js

```diff
@@ -48,6 +49,11 @@ module.exports = opts => {
 	});
 
 	app.on('ready', () => {
+		// activate devtron for the user if they have it installed
+		try {
```

Comment:
It's not because one could not want it when it is installed, it's because you are swallowing all exception that could be thrown by the code block. This could make difficult to debug problems...